lineage/states/stateCommon.py

```diff
@@ -12,16 +12,15 @@ def bern_pdf(x, p):
     probability distribution function.
     """
     # bern_ll = self.bern_p**(tuple_of_obs[0]) * (1.0-self.bern_p)**(1-tuple_of_obs[0])
-
+    
     return (p**x) * ((1.0 - p)**(1 - x))
 
 
-def bernoulli_estimator(bern_obs):
+def bernoulli_estimator(bern_obs, old_params, gammas):
```

Comment:
Is `old_params` used?

Reply:
It's not in this estimator, and not in the exponential one either, but it is in the gamma estimator. I wanted it there so that all the estimators follow the same argument line:
1. observations
1.5 censor observations (for the time distributions)
2. a tuple of old params,
3. gammas

I can remove it if it's better that it's not there.

Reply:
Ok. Good as is.